examples/main/views/click.js

```diff
@@ -20,44 +20,37 @@
 
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import MapGL, {autobind} from 'react-map-gl';
+import {InteractiveMap, experimental} from 'react-map-gl';
 /* global window */
 
 const windowAlert = window.alert;
+const autobind = experimental.autobind;
```

Comment:
I personally don't think autobind should be part of react-map-gl's experimental exports, and I don't think that our examples should be including any experimental exports (unless that is the topic of the example). We should write code in these examples that use standard coding practices. I'd probably just bind in the constructor to make things simple and standard ES6 compatible.